### PR TITLE
Chore: Fix prettier issues in test files

### LINKED ISSUE #1709: make lint is failing on v4.4.2/v4.4.3
**Describe the bug**
Running `make lint` after building v4.4.2 is failing with the below errors (it works as expected against v4.4.1 though).

**Expected behavior**
Functional lint test.

**Logs**
```
yarn lint
yarn run v1.22.19
$ eslint --cache --ignore-path ./.gitignore --ext .js,.jsx,.ts,.tsx .

/build/grafana-zabbix/src/grafana-zabbix-4.4.2/src/datasource/specs/dbConnector.test.ts
   9:24  error  Insert `,`  prettier/prettier
  20:34  error  Insert `,`  prettier/prettier

/build/grafana-zabbix/src/grafana-zabbix-4.4.2/src/datasource/specs/timeseries.spec.ts
   5:1   error  Delete `⏎`                                                                                                  prettier/prettier
   9:10  error  Replace `[0,·1],·[1,·2],·[1,·3]` with `⏎··········[0,·1],⏎··········[1,·2],⏎··········[1,·3],⏎········`     prettier/prettier
  10:10  error  Replace `[2,·1],·[3,·2],·[4,·3]]` with `⏎··········[2,·1],⏎··········[3,·2],⏎··········[4,·3],⏎········],`  prettier/prettier
  13:23  error  Replace `[2,·1],·[4,·2],·[5,·3]` with `⏎········[2,·1],⏎········[4,·2],⏎········[5,·3],⏎······`             prettier/prettier
  23:10  error  Replace `[1,·1],·[1,·2],·[1,·3]` with `⏎··········[1,·1],⏎··········[1,·2],⏎··········[1,·3],⏎········`     prettier/prettier
  24:10  error  Replace `[3,·2],·[4,·3]]` with `⏎··········[3,·2],⏎··········[4,·3],⏎········],`                            prettier/prettier
  27:23  error  Replace `[1,·1],·[4,·2],·[5,·3]` with `⏎········[1,·1],⏎········[4,·2],⏎········[5,·3],⏎······`             prettier/prettier
  35:21  error  Replace `[1,·1],·[-4,·2],·[2,·3]` with `⏎········[1,·1],⏎········[-4,·2],⏎········[2,·3],⏎······`           prettier/prettier
  37:23  error  Replace `[101,·1],·[96,·2],·[102,·3]` with `⏎········[101,·1],⏎········[96,·2],⏎········[102,·3],⏎······`   prettier/prettier

/build/grafana-zabbix/src/grafana-zabbix-4.4.2/src/datasource/specs/utils.spec.ts
    5:1   error  Replace `⏎··describe('expandItemName()',·()·=>·{⏎` with `··describe('expandItemName()',·()·=>·{`                                                                                                                   prettier/prettier
   13:21  error  Replace `"CPU·user·time"` with `'CPU·user·time',`                                                                                                                                                                  prettier/prettier
   18:21  error  Replace `"CPU·system·time·-·avg1"` with `'CPU·system·time·-·avg1',`                                                                                                                                                prettier/prettier
   23:21  error  Replace `"CPU·-··-·system·-·avg1"` with `'CPU·-··-·system·-·avg1',`                                                                                                                                                prettier/prettier
   24:10  error  Insert `,`                                                                                                                                                                                                         prettier/prettier
   27:26  error  Replace `test_case` with `(test_case)`                                                                                                                                                                             prettier/prettier
   39:21  error  Replace `"CPU·user·time"` with `'CPU·user·time',`                                                                                                                                                                  prettier/prettier
   44:21  error  Replace `"CPU·type=user,value=avg·time"` with `'CPU·type=user,value=avg·time',`                                                                                                                                    prettier/prettier
   49:21  error  Replace `"CPU·type=user,value=avg·time·user"` with `'CPU·type=user,value=avg·time·user',`                                                                                                                          prettier/prettier
   54:21  error  Replace `"CPU·type=user,value=avg·time·user"` with `'CPU·type=user,value=avg·time·user',`                                                                                                                          prettier/prettier
   55:10  error  Insert `,`                                                                                                                                                                                                         prettier/prettier
   58:26  error  Replace `test_case` with `(test_case)`                                                                                                                                                                             prettier/prettier
   70:21  error  Replace `"CPU·user,system·-·avg1·time"` with `'CPU·user,system·-·avg1·time',`                                                                                                                                      prettier/prettier
   75:59  error  Insert `,`                                                                                                                                                                                                         prettier/prettier
   80:59  error  Insert `,`                                                                                                                                                                                                         prettier/prettier
   81:10  error  Insert `,`                                                                                                                                                                                                         prettier/prettier
   84:26  error  Replace `test_case` with `(test_case)`                                                                                                                                                                             prettier/prettier
   92:43  error  Delete `⏎`                                                                                                                                                                                                         prettier/prettier
   99:63  error  Replace `\w|\d)*-alu-[0-9{2}/']` with `w|d)*-alu-[0-9{2}/'],`                                                                                                                                                      prettier/prettier
  103:41  error  Insert `,`                                                                                                                                                                                                         prettier/prettier
  104:10  error  Insert `,`                                                                                                                                                                                                         prettier/prettier
  107:26  error  Replace `test_case` with `(test_case)`                                                                                                                                                                             prettier/prettier
  118:61  error  Replace `\w|\d)*-alu-[0-9]*/']` with `w|d)*-alu-[0-9]*/'],`                                                                                                                                                        prettier/prettier
  122:61  error  Replace `\w|\d)*-alu-[0-9]{2}/']` with `w|d)*-alu-[0-9]{2}/'],`                                                                                                                                                    prettier/prettier
  126:41  error  Insert `,`                                                                                                                                                                                                         prettier/prettier
  130:35  error  Insert `,`                                                                                                                                                                                                         prettier/prettier
  131:10  error  Insert `,`                                                                                                                                                                                                         prettier/prettier
  134:26  error  Replace `test_case` with `(test_case)`                                                                                                                                                                             prettier/prettier
  147:19  error  Insert `,`                                                                                                                                                                                                         prettier/prettier
  151:19  error  Insert `,`                                                                                                                                                                                                         prettier/prettier
  154:19  error  Replace `[1,·2],·[3,·4]` with `⏎············[1,·2],⏎············[3,·4],⏎··········`                                                                                                                                prettier/prettier
  155:19  error  Insert `,`                                                                                                                                                                                                         prettier/prettier
  158:19  error  Replace `[[1,·2],·[3,·4]],·[[1,·2],·[3,·4]]` with `⏎············[⏎··············[1,·2],⏎··············[3,·4],⏎············],⏎············[⏎··············[1,·2],⏎··············[3,·4],⏎············],⏎··········`  prettier/prettier
  159:19  error  Insert `,`                                                                                                                                                                                                         prettier/prettier

/build/grafana-zabbix/src/grafana-zabbix-4.4.2/src/datasource/zabbix/connectors/zabbix_api/types.ts
  22:37  error  Replace `[key:·string]:·any` with `·[key:·string]:·any·`  prettier/prettier
  26:33  error  Replace `[key:·string]:·any` with `·[key:·string]:·any·`  prettier/prettier

✖ 47 problems (47 errors, 0 warnings)
  47 errors and 0 warnings potentially fixable with the `--fix` option.

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
make: *** [Makefile:72: lint] Error 1
```

I remain available if you need any more information :)

### PR DESCRIPTION
**Pull request by zoltanbedi**
Fixes #1709

### CODE REVIEW

**Comment by Antiz96** [comment]
I confirm https://github.com/grafana/grafana-zabbix/pull/1710/commits/54612e6d5398a78d7f25b3403f311aa21d819034 fixes the issue. Thanks @zoltanbedi!

**Comment by Antiz96** [comment]
@zoltanbedi Well, I'm at least waiting for this to be reviewed/merged (just by conscience) to package the new version on Arch Linux repo.
If there are plan to release a `v4.4.4` including this fix I can wait for it, otherwise I'm able to simply backport the fixing commit in our `v4.4.3` package :)

File: src/datasource/specs/utils.spec.ts
Comment (gabor) [inline src/datasource/specs/utils.spec.ts:99]: i think something might be wrong here somehow..

i mean, i think the prettier-transform is correct.

but, the `\w\d` part looks very regex-ish, so maybe the test was incorrect already?

File: src/datasource/specs/utils.spec.ts
Comment (gabor) [inline src/datasource/specs/utils.spec.ts:96]: `\w\d` got turned into `wd`. i think this is a correct prettier-transform, but the original text looks very regex-ish... maybe the "before" situation was already wrong?

Comment (zoltanbedi) [reply src/datasource/specs/utils.spec.ts:96]: you are right the query to escape w properly needs double backslash.

**Comment by zoltanbedi** [comment]
@Antiz96 if you are not blocked then we won't cut a release for code formatting. We will do that once we have other things to release as well.

**Comment by Antiz96** [comment]
@zoltanbedi Sure, sounds good to me. I'll just backport the fixing commit once this PR is merged :)

**Review by gabor-grafana — APPROVED** [review]
LGTM 👍

**Pull request merged**
